versioned_docs/version-3.0.0-beta/env-var.md

```diff
@@ -0,0 +1,54 @@
+---
+id: env-var
+title: Environment variable
+sidebar_label: Environment variable
```

Comment:
There is already a page for Env var inside the project category. We don't need this page here. Please transfer all the content to that page

Reply:
https://docs.spheron.network/projects/environment-variables